sketch_esp8266/src/Patterns/Seahawks.h

```diff
@@ -0,0 +1,69 @@
+#ifndef Seahawks_h
+#define Seahawks_h
+
+#include "Arduino.h"
+#include <FastLED.h>
+#include "CelebrationPattern.h"
+#include "LedDisplayConf.h"
+
+class Seahawks : public CelebrationPattern
+{
+  private:
+    uint8_t _i;
+    uint8_t _x;
+
+  public:
+    Seahawks()
+    {
+      _i = 0;
+      _x = 0;
+      m_speed = 1;
+    };
+
+    bool draw(CRGB leds[NUM_STRIPS][NUM_LEDS_PER_STRIP], bool someoneJustScored = false)
+    {
+      // reset your pattern to play from the begining if someoneJustScored
+      if (someoneJustScored)
+      {
+        _i = -1;
+        _x = 0;
+      }
+
+      // restore state (try to limit the amount of state you store... try to recover state algorithmically)
+      uint8_t i = _i;
+      uint8_t x = _x;
+
+      i++;
+
+      if (i == NUM_LEDS_PER_STRIP)
+      {
+        i = 0;
+        x++;
+      }
+
+      // have a way to know when your pattern is complete.
+      // and if pattern is completed do nothing
+      // return true
+      if (_x == NUM_STRIPS)
+      {
+        return true;
+      }
+      CRGB dodgerBlue = CRGB::DodgerBlue;
+      CRGB lawnGreen = CRGB::LawnGreen;
+      leds[x][i] = lawnGreen;
+      leds[x + 1][i] = dodgerBlue;
+      leds[x + 2][i] = lawnGreen;
+      leds[x + 3][i] = dodgerBlue;
+      leds[x + 4][i] = lawnGreen;
+      leds[x + 5][i] = dodgerBlue;
+      m_speed -= 0.1;
+  
+      _i = i;
+      _x = x;
+
+      return false;
+    }
+
+};
+
+#endif
```

Comment:
```suggestion
#ifndef Seahawks_h
#define Seahawks_h

#include "Arduino.h"
#include <FastLED.h>
#include "CelebrationPattern.h"
#include "LedDisplayConf.h"

class Seahawks : public CelebrationPattern
{
 
  public:
    Seahawks()
    {
      m_speed = 60;
    };

    bool draw(CRGB leds[NUM_STRIPS][NUM_LEDS_PER_STRIP], bool someoneJustScored = false)
    {

    static uint8_t i =0;
    static uint8_t x = 0;

      // reset your pattern to play from the begining if someoneJustScored
      if (someoneJustScored)
      {
        i = -1;
        x = 0;
      }

      i++;

      if (i == NUM_LEDS_PER_STRIP)
      {
        i = 0;
        x++;
      }

      // have a way to know when your pattern is complete.
      // and if pattern is completed do nothing
      // return true
      if (x == NUM_STRIPS)
      {
        return true;
      }

      leds[x][i] = CRGB::LawnGreen;
      leds[x + 1][i] = CRGB::DodgerBlue;
      leds[x + 2][i] = CRGB::LawnGreen;
      leds[x + 3][i] = CRGB::DodgerBlue;
      leds[x + 4][i] = CRGB::LawnGreen;
      leds[x + 5][i] = CRGB::DodgerBlue;
      m_speed -= 1;
  
    
      return false;
    }

};

#endif
```

Reply:
since you are just drawing all the led strips at once, you dont need an `x`. Just use `i` from 0 to 59...